jboss-kie-wildfly-common/module.yaml

```diff
@@ -25,6 +25,9 @@ envs:
       example: "distinguishedName"
     - name: "AUTH_LDAP_JAAS_SECURITY_DOMAIN"
       description: "he JMX ObjectName of the JaasSecurityDomain used to decrypt the password."
+    - name: "AUTH_LDAP_FLAG"
```

Comment:
I think I asked you to rename to AUTH_LDAP_LOGIN_MODULE, then you rename and now renamed back, is there a reason?